config/locales/en.yml

```diff
@@ -2020,26 +2020,26 @@ en:
           (public information of Slovenia).
         contributors_si_slovenia: Slovenia
         contributors_si_gu: Surveying and Mapping Authority
-        contributors_si_gu_url: http://www.gu.gov.si/en/
+        contributors_si_gu_url: https://www.gov.si/en/state-authorities/bodies-within-ministries/surveying-and-mapping-authority/ 
         contributors_si_mkgp: Ministry of Agriculture, Forestry and Food
-        contributors_si_mkgp_url: http://www.mkgp.gov.si/en/
+        contributors_si_mkgp_url: https://www.gov.si/en/state-authorities/ministries/ministry-of-agriculture-forestry-and-food/
         contributors_es_credit_html: |
           %{spain}: Contains data sourced from the
           Spanish National Geographic Institute (%{ign_link}) and
           National Cartographic System (%{scne_link})
           licensed for reuse under %{cc_by_link}.
         contributors_es_spain: Spain
         contributors_es_ign: IGN
-        contributors_es_ign_url: http://www.ign.es/
+        contributors_es_ign_url: https://www.ign.es/
         contributors_es_scne: SCNE
-        contributors_es_scne_url: http://www.scne.es/
+        contributors_es_scne_url: https://www.scne.es/
         contributors_es_cc_by: CC BY 4.0
         contributors_es_cc_by_url: https://creativecommons.org/licenses/by/4.0/
         contributors_za_credit_html: |
           %{south_africa}: Contains data sourced from %{ngi_link}, State copyright reserved.
         contributors_za_south_africa: South Africa
         contributors_za_ngi: "Chief Directorate: National Geo-Spatial Information"
-        contributors_za_ngi_url: http://www.ngi.gov.za/
+        contributors_za_ngi_url: https://www.ngi.gov.za/
```

Comment:
That URL results in a certificate warning. Did you mean to use https://ngi.dalrrd.gov.za/ instead? That is where the http one redirects to?

Reply:
Yes, https://ngi.dalrrd.gov.za/ is now the correct URL for NGI.

Reply:
Thanks, fixed.